transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/index.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    How to Apply For a Federal Job with the CFPB
+{%- endblock %}
+
+{% block desc -%}
+    Transcript for the video "How to Apply For a Federal Job with the CFPB."
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    transcripts
```

Comment:
I'll just remove for now. We actually don't need it, but I expect if we do we'll make the changes across all the transcripts. We have a "About the Bureau" transcript to add next.